applications/ConstitutiveLawsApplication/custom_constitutive/plane_stress_d_plus_d_minus_damage_masonry_2d.cpp

```diff
@@ -883,11 +883,11 @@ void DamageDPlusDMinusMasonry2DLaw::EvaluateBezierCurve(
 		bezier_law_param_C 	= x1 - xi;
 	}
 
-	double bezier_law_param_D = bezier_law_param_B * bezier_law_param_B - 
+	double bezier_law_param_D = bezier_law_param_B * bezier_law_param_B -
```

Comment:
can we add const here?

Reply:
probably but this should be a different PR